docs/success-stories/success-story-2/index.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: success-story-page
+show_sidebar: false
+permalink: /success-stories/success-story-Developing-Machine-Learning-Tools-Through-Collaborative-Annotation-Studies/
+
+# Banner Content
+# =================================
+title: Success Stories
+subtitle: Developing machine-learning tools through collaborative annotation studies
+hero_image: /assets/img/home-callouts/success_stories.jpg
+
+# Story
+# =================================
+story:
+ - title: Developing Machine-Mearning Tools Through Collaborative Annotation Studies
+   link-id: Developing-Machine-Learning-Tools-Through-Collaborative-Annotation-Studies
+   description-para-1: "The primary determinant in machine learning algorithm performance is the availability of abundant labeled data for training. Collecting data for applications like tissue region segmentation and cell classification is challenging given the limited availability of experts and tools needed to collect and review annotations.<br><br>
+   Our web-based platform enables users to collaborate all over the world, and has been used to generate over 120,000 human markups of histopathology in multiple annotation studies."
+   description-para-2: "Our API and user roles allow us to create collaborative annotation study teams that engage users with different expertise levels, ranging from pathologists to medical students, to collaboratively generate large and extensively reviewed annotation datasets. In a study with over 25 participants from over 5 countries, we generated over 25,000 annotated tissue regions to generate the richest public dataset of annotated breast-cancer tissues to date. The ability to programmatically monitor and manage these studies through the API is key to their success.<br><br>
+   Collecting data on this scale allows us to generate highly accurate machine-learning models for tasks like tissue segmentation and cell classification and detection, and to understand the limits of concordance among human experts.<br><br>
+   Link to Kitware demo site	- http://demo.kitware.com/histomicstk/"
+   resource-link: http://demo.kitware.com/histomicstk/
```

Comment:
At least on the circle-ci artifacts, this isn't being rendered as a link.

Reply:
(There is a Resource button, but I expected the URL to be a link).

Reply:
@manthey 
do you suggest removing the button and make url a link, or keep the button too? 
coz button and url will be linking to same thing.

Reply:
I don't really have a preference if the button text can be easily change.  I expected the url to be a link; it seemed like it should be a different location than something called "View Resource".  I think that you could get rid of the text "Link to Kitware demo site" and just have the button if the button text was something like "View Demo Site".

Reply:
fixed in a53066c0783d2fceb59f91712980f5d090664572